modules/virt-configuring-sriov-networks-for-vms.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-defining-an-sriov-network.adoc
+
+// This is similar to the OpenShift `configuring-sr-iov-networks` module,
```

Comment:
Is this so dissimilar that we need a distinct copy? What happens when they begin to diverge, as undoubtedly will happen. I've been working through multiple reworks of all the SR-IOV Network Operator content:

- https://github.com/openshift/openshift-docs/pull/22512
- https://github.com/openshift/openshift-docs/pull/22513

Reply:
@jboxman I anticipated larger deviation for virtualization than was required. I'm still going through the SME feedback for this at the moment but it looks like there are no key differences. I'll need to be pretty liberal with the conditionals (CNV wants a minimal approach for now) but if you're happy with that then I'll do that instead.